Index.html

```diff
@@ -20,21 +20,159 @@
+                <li><a href="#">2015</a></li> 
+                          
+             </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+      <div><a href="" class="seeproject">See Project</a></div>
+    </section>
+    <section class="section">
+        <span>&nbsp;</span>
+        <span>&nbsp;</span>
+        <h2>About Myself</h2>
+        
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+     
+    </section>
+    
+    <section class="section">
+        <span class="connect"><a href="#" > LET’S CONNECT</a></span>
+        <ul class="socials">
            <li><a href="#"><img src="./images/twitter.png"></a></li>
            <li><a href="#"><img src="./images/linkedin.svg"></li>
            <li><a href="#"><img src="./images/m.png"></a></li>
            <li><a href="#"><img src="./images/git.png"></a></li>
            <li><a href="#"><img src="./images/peace.png"></a></li>
         </ul>
-    </footer>
+        <div><a href="" class="seeproject">Get my resume</a></div>
+    </section>
+    <section class="section">   
+        <h3>Languages</h3>
+        <ul class="tech">
+            <li><a href="#"><img src="./images/js.png"></li>
+            <li><a href="#"><img src="./images/html.png"></li>
+            <li><a href="#"><img src="./images/css.png"></li>    
+                <li><a href="#"><img src="./images/frame.png"></li>
+                    <li><a href="#"><img src="./images/skills.png"></li>         
+        </ul>
+    </section>
+    <section class="contact">   
+        <h1 >Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+        <form class="pure-form">
+
+            
+            <input type="text" 
+                class="input-rounded" placeholder="Lorem Ipsum">
+              
+            
+            <input type="text" 
+            class="input-rounded" placeholder="loremipsum@gmail.com">
+                  
+           
+
+            <textarea name="comment" >Write your message here</textarea>
+
+            <input type="submit" class="submit-button" value="Get in touch"/>
+        </form>
+
+    </section>
```

Comment:
- Kindly remove the contact form as it has not yet been part of the requirement you will work on it in the next project :+1: